source/extensions/filters/http/health_check/health_check.h

```diff
@@ -17,11 +20,43 @@ namespace HttpFilters {
 namespace HealthCheck {
 
 /**
- * Shared cache manager used by all instances of a health check filter configuration as well as
- * all threads. This sets up a timer that will invalidate the cached response code and allow some
- * requests to go through to the backend. No attempt is made to allow only a single request to go
- * through to the backend, so during the invalidation window some number of requests will get
- * through.
```

Comment:
Why was this comment (for the HealthCheckCacheManager) removed?

Reply:
Sorry, when I was refactoring I had a TODO to copy/move the comments, I pushed too early. I'll add them back.